content/pages/download-va-letters/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Download VA Letters and Documents
+layout: page-breadcrumbs.html
+template: level2-index
+---
+
+<main>
```

Comment:
It turns out `<main>` is [not IE11-friendly](https://caniuse.com/#feat=html5semantic). In that case, let's stick with `<div id="main">`, sorry to ask you to revert this. Also, it seems like the `<section>` tag immediately within that is not necessary, so let's go ahead and remove that.